components/studio/test/shared/studio-enter.exp

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env expect
+
+set env(HAB_NOCOLORING) true
+set studio_enter_command $::env(studio_enter_command)
```

Comment:
It's conventional for variables that will be exported into the environment to be all caps. Any reason not to make this `STUDIO_ENTER_COMMAND`?

Reply:
No reason to not make the change, it's left over from an earlier iteration. Will update.